packages/amplify-category-geo/src/service-utils/deviceLocationTrackingParams.ts

```diff
@@ -0,0 +1,35 @@
+import _ from 'lodash';
+import { printer } from 'amplify-prompts';
+import { ResourceParameters } from './resourceParams';
+
+/**
+ * DeviceLocationTrackingParameters
+ */
+export type DeviceLocationTrackingParameters = ResourceParameters & {
+  groupPermissions: string[];
+  roleAndGroupPermissionsMap: Record<string, string[]>;
+};
+
+/**
+ * isCompleteTrackingParams
+ */
+export const isCompleteTrackingParams = (
+  partial: Partial<DeviceLocationTrackingParameters>,
+): partial is DeviceLocationTrackingParameters => {
+  printer.info(`params: ${JSON.stringify(partial)}`);
```

Comment:
Has this gone through an API review? Why not use formatting when printing this information?

Reply:
whoops this was included by mistake

Reply:
But yep! We did an API review for the whole process, this print out of the params was not included of course